### PR TITLE
goliath tentacles chainstunning will only increase the current stun by 2 seocnds instead of resetting it to 10

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

Regardless of the contents of a change to this it is clear that goliath chainstuns are unfun to get hit by and anyone who disagrees is wrong and bad

### Why is this change good for the game?
attempts to fix the same problem #11008 does but in a different way
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Goliath tentacles will only stun for 2 seconds if the person getting hit is already stunned, if they are not stunned, the tentacles will stun for 10 seconds
### What general grouping does this PR fall under? 
mining

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

# Changelog

:cl:  
tweak: goliath tentacles will only increase existing stuns by 2 seconds instead of setting them to 10 to reduce chainstunning potential
/:cl:

### CODE REVIEW

**Comment by ynot01** [comment]
an interesting alt, ill let the maints decide which is better, though i still dislike stun mechanics

**Comment by ChesterTheCheesy** [comment]
This is much better

**Comment by N3D6** [comment]
Yeah this is pretty good too, goliaths chainstunning is pretty shitty, it happening for a lower amount of time seems a lot better.

**Comment by ToasterBiome** [comment]
I approve of this, more forgiving but still keeps what i think is a critical goliath ability

**Comment by LazennG** [comment]
something something miners op so pr bad
